### ISSUE TITLE
Reduce the Docker image size

### ISSUE THREAD

**Issue by jspenger** [portals-project/portals#233]
The current Docker image for the Portals Client/Server is about 1.3 GB. This is very large and wasteful, and can likely be reduced down to a much smaller size (~100MB). As this is very common, there should also be plenty of other examples on the web on how to do this, for example a blog post like this: https://medium.com/jeroen-rosenberg/lightweight-docker-containers-for-scala-apps-11b99cf1a666.

To summarize, the task is to reduce the Docker image size for the Portals Client/Server.
